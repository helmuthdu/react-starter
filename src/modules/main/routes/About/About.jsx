// @flow
import React from 'react';

const About = () => {
  return (
    <section>
      <p>This project is created by @helmuthdu.</p>
      <p>
        For any question, Im on{' '}
        <a href="mailto:helmuthdu@gmail.com" target="blank">
          helmuthdu@gmail.com
        </a>
      </p>
      <p>
        For any issues, any PR are welcoming
        <a href="https://github.com/helmuthdu/react-start" target="blank">
          {' '}
          on this repository
        </a>
      </p>
    </section>
  );
};

export default About;
