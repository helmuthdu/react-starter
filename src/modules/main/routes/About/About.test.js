import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import React from 'react';
import About from './About';

describe('Route -> About component', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<About />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
