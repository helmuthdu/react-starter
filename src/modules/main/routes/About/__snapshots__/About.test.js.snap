// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Route -> About component renders without crashing 1`] = `
<section>
  <p>
    This project is created by @helmuthdu.
  </p>
  <p>
    For any question, I'm on
     
    <a
      href="mailto:helmuthdu@gmail.com"
      target="blank"
    >
      helmuthdu@gmail.com
    </a>
  </p>
  <p>
    For any issues, any PR are welcoming
    <a
      href="https://github.com/helmuthdu/react-start"
      target="blank"
    >
       
      on this repository
    </a>
  </p>
</section>
`;
